Zend/tests/bug43332_1.phpt

```diff
@@ -12,7 +12,7 @@ $foo = new foo;
 $foo->bar($foo); // Ok!
 $foo->bar(new \stdclass); // Error, ok!
 --EXPECTF--
-Fatal error: Uncaught TypeError: Argument 1 passed to foobar\foo::bar() must be an instance of foobar\foo, instance of stdClass given, called in %sbug43332_1.php on line 10 and defined in %sbug43332_1.php:5
+Fatal error: Uncaught TypeError: foobar\foo::bar() expects argument 1 to be of type self, stdClass given, called in %sbug43332_1.php on line 10 and defined in %sbug43332_1.php:5
```

Comment:
Apparently `zend_type_to_string()` doesn't translate `self` to the actual class... (I wanted to highlight this notable change even though it makes sense for me)

Reply:
You can use `zend_type_to_string_resolved()` for this.